packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/useApprovalRequestActionsCell.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import { useEntityUserByAuthToken } from '@/core/queries';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import CheckCircleOutlineRoundedIcon from '@mui/icons-material/CheckCircleOutlineRounded';
+import HighlightOffRoundedIcon from '@mui/icons-material/HighlightOffRounded';
+import { Stack } from '@mui/material';
+import { IconButton } from '@mui/material';
+import { GridBaseColDef } from '@mui/x-data-grid/models/colDef/gridColDef';
+
+type ApprovalRequestsTableRowAction = 'approve' | 'reject';
+
+export interface UseApprovalRequestActionsCellProps {
+  /**
+   * The event handler for an approval row action.
+   * If not provided, the action menu will not be displayed.
+   * @param props.id - The identifier of the clicked row, a string.
+   * @param props.action - The action to be performed, an `InvoicesTableRowAction`.
+   */
+  onRowActionClick: (props: {
+    id: string;
+    action?: ApprovalRequestsTableRowAction;
```

Comment:
It seems that you forgot to remove `?` in our case `action` will always be present (the example from the source was in error).

Reply:
fixed